core/src/storage/resident_credentials.py

```
@@ -17,23 +17,53 @@ def get(index: int) -> bytes | None:
     return common.get(common.APP_WEBAUTHN, index + _RESIDENT_CREDENTIAL_START_KEY)
 
 
-def set(index: int, data: bytes) -> None:
+def set(index: int, data: bytes, is_overwritten: bool = False) -> None:
```

Ruff (0.8.2)

Boolean-typed positional argument in function definition

(FBT001)